### DISCUSSION TITLE
No Wi-Fi strength for sensors in the app

### DISCUSSION THREAD

**Discussion by daxyorg** [Flo-Schilli/homeassistant-grohe_sense#45] [Q&A] [answered]
Hi community!

Does anyone have the following problem - in the phone application does not visualize the wifi signal only for the sensors?
I'm using iPhone, last app version (2.2.1, build 3).
Version for sensors: 01.20

<img width="300" alt="Screenshot" src="https://github.com/user-attachments/assets/432401d6-229c-4d22-b46e-725c05bcd70b" />

**Comment by Flo-Schilli** [comment, marked as answer]
It's looking the same for me 🤷

**Reply by daxyorg** [reply]
Thanks for this confirmation!

**Reply by Flo-Schilli** [reply]
You're welcome :smile:
